### PR TITLE
Enhanced article "Use Playwright to automate and test in Microsoft Edge"

### PR DESCRIPTION
**Pull request by mxschmitt**
**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/playwright/?branch=pr-en-us-1519

**Before:** 
https://docs.microsoft.com/en-us/microsoft-edge/playwright/

### CODE REVIEW

File: microsoft-edge/playwright/index.md [hunk 1]
```diff
@@ -11,110 +11,130 @@ keywords: microsoft edge, web development, developer, tools, automation, test, p
...
+`example.js` is a simple demonstration of the automation and testing scenarios that are enabled by Playwright.  To take screenshots in other web browsers, change the above code from `await playwright.chromium.launch` to the following code:
 
-[npmEdgePaths]: https://www.npmjs.com/package/edge-paths "edge-paths | npm"
+Firefox: 
 
-[PlaywrightMain]: https://playwright.dev "Playwright"
-[PlaywrightAPIReference]: https://playwright.dev#?path=docs/api.md "Playwright API Reference"
-[PlaywrightAPIPageSetViewport]: https://playwright.dev#?path=docs%2Fapi.md&q=pagesetviewportsizeviewportsize "page.setViewportSize(viewportSize) | Playwright API Reference"
-[PlaywrightSystemRequirements]: https://playwright.dev#?path=docs/intro.md&q=system-requirements "Playwright System Requirements"
+```javascript
```
Comment (mikehoffms) [inline microsoft-edge/playwright/index.md:130]: @mxschmitt - please tech-review this code and the WebKit code below it.

Comment (mxschmitt) [reply microsoft-edge/playwright/index.md:130]: looks good!

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/playwright/index.md [hunk 2]
```diff
@@ -86,7 +124,7 @@ Playwright launches Microsoft Edge, navigates to `https://www.microsoft.com/edge
 *   Firefox  `await firefox.launch()`
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply microsoft-edge/playwright/index.md]: LGTM. Thanks!

File: microsoft-edge/playwright/index.md [hunk 3]
```diff
@@ -11,110 +11,130 @@ keywords: microsoft edge, web development, developer, tools, automation, test, p
...
 
-Playwright launches [headless browsers][WikiHeadlessBrowser] by default.  Headless browsers do not display a UI, so instead you must use the command line.  You may also configure Playwright to run full \(non-headless\) Microsoft Edge as well.
+Playwright launches [headless browsers](https://en.wikipedia.org/wiki/Headless_browser) by default.  Headless browsers don't display a UI, so instead you must use the command line.  You can also configure Playwright to run full (non-headless) Microsoft Edge as well.
 
-By default, when you install Playwright, the installer downloads [Chromium][ChromiumHome], [Firefox][FirefoxMain], and [WebKit][WebKitMain].  If you have Microsoft Edge installed as well, Playwright just needs a one-line code change to test your website or app in Microsoft Edge.  To download Microsoft Edge, navigate to [Download Microsoft Edge][MicrosoftEdgeDownload].
 
-## Installing Playwright
+<!-- ====================================================================== -->
+## Install Playwright and browser binaries
```
Comment (mxschmitt) [inline microsoft-edge/playwright/index.md:20]:
```suggestion
## Install Playwright and browsers
```

File: microsoft-edge/playwright/index.md [hunk 4]
```diff
@@ -11,110 +11,130 @@ keywords: microsoft edge, web development, developer, tools, automation, test, p
...
+export default config
 ```
 
-Finally, set `executablePath: EDGE_PATH` in `example.js`.  Save your changes.
-
-> [!NOTE]
-> Microsoft Edge \(EdgeHTML\) doesn't work with Playwright.  You must install [Microsoft Edge \(Chromium\)][MicrosoftEdgeDownload] to continue following this example.
-
-Now run `example.js` from the command line.
+If Microsoft Edge isn't already installed on your system, install it through Playwright, as follows:
```
Comment (mxschmitt) [inline microsoft-edge/playwright/index.md:89]:
```suggestion
If Microsoft Edge Beta isn't already installed on your system, install it through Playwright, as follows:
```

**Comment by mxschmitt** [comment]
> @mxschmitt - is this PR ready to merge/publish?

Yes my peer aslushnikov approved, so we'd have double safety. I added two more nit comments, otherwise it looks good from our side. Thanks for the fast review! :)
